bot/code_review_bot/workflow.py

```diff
@@ -322,6 +334,11 @@ def _in_group(dep_id):
                     continue
                 artifacts = task.load_artifacts(self.queue_service)
                 if artifacts is not None:
+                    if "public/firefox-source-docs-url.txt" in artifacts:
```

Comment:
I think you should declare "public/firefox-source-docs-url.txt" once as a member
in case the file name changes, we will need to update it only twice